strings/french.strings

```diff
@@ -208,1285 +208,1285 @@
 
-/* MISSING TRANSLATION - views/pages/Home.vue */
-"Signup" = "Signup";
+/* views/pages/Home.vue */
+"Signup" = "Créer un compte";
 
-/* MISSING TRANSLATION - views/components/group-creation-steps/GroupName.vue */
-"Upload an image" = "Upload an image";
+/* views/components/group-creation-steps/GroupName.vue */
+"Upload an image" = "Uploader une image";
 
-/* MISSING TRANSLATION - views/pages/Mailbox.vue */
-"No Messages" = "No Messages";
+/* views/pages/Mailbox.vue */
+"No Messages" = "Pas de Messages";
 
-/* MISSING TRANSLATION - views/containers/proposals/ProposalsWidget.vue, views/pages/DesignSystem.vue */
-"No one has created a proposal yet." = "No one has created a proposal yet.";
+/* views/containers/proposals/ProposalsWidget.vue, views/pages/DesignSystem.vue */
+"No one has created a proposal yet." = "Personne n'a encore créé de proposition.";
 
-/* MISSING TRANSLATION - views/containers/contributions/ContributionsWidget.vue, views/containers/navigation/Navigation.vue, views/pages/Contributions.vue */
+/* views/containers/contributions/ContributionsWidget.vue, views/containers/navigation/Navigation.vue, views/pages/Contributions.vue */
 "Contributions" = "Contributions";
 
-/* MISSING TRANSLATION - views/containers/group-settings/GroupLeaveModal.vue */
-"LEAVE {GROUP_NAME}" = "LEAVE {GROUP_NAME}";
+/* views/containers/group-settings/GroupLeaveModal.vue */
+"LEAVE {GROUP_NAME}" = "QUITTER {GROUP_NAME}";
 
-/* MISSING TRANSLATION - views/containers/payments/PaymentRowReceived.vue, views/containers/payments/PaymentRowSent.vue, views/containers/payments/PaymentRowTodo.vue */
-"Not received" = "Not received";
+/* views/containers/payments/PaymentRowReceived.vue, views/containers/payments/PaymentRowSent.vue, views/containers/payments/PaymentRowTodo.vue */
+"Not received" = "Non reçu";
 
-/* MISSING TRANSLATION - views/containers/user-settings/Troubleshooting.vue */
-"Downloading new data..." = "Downloading new data...";
+/* views/containers/user-settings/Troubleshooting.vue */
+"Downloading new data..." = "Téléchargement des nouvelles données...";
 
-/* MISSING TRANSLATION - views/containers/proposals/ProposalItem.vue */
-"TODO: Change [generic] from [current] to [new-value]" = "TODO: Change [generic] from [current] to [new-value]";
+/* views/containers/proposals/ProposalItem.vue */
+"TODO: Change [generic] from [current] to [new-value]" = "TODO: Changer [generic] de [current] à [new-value]";
 
-/* MISSING TRANSLATION - views/pages/Payments.vue */
-"You haven’t received any payments yet" = "You haven’t received any payments yet";
+/* views/pages/Payments.vue */
+"You haven’t received any payments yet" = "Vous n'avez pas encore reçu de paiements";
 
-/* MISSING TRANSLATION - views/pages/Mailbox.vue */
+/* views/pages/Mailbox.vue */
 "Menu" = "Menu";
 
-/* MISSING TRANSLATION - views/pages/GroupChat.vue */
-"Search for a channel" = "Search for a channel";
+/* views/pages/GroupChat.vue */
+"Search for a channel" = "Rechercher un canal";
```

Comment:
Agreed on adding "de discussion".

Along the way, this also made me think about using "canal de discussion",
mainly to avoid any confusion between searching for just a group chat and searching for a group of users.

Which alternative do you prefer?
- `Rechercher un groupe de discussion`
- `Rechercher un canal de discussion`

Reply:
I personally don't understand canal de discussion.. maybe I've been out of France for too long?

Reply:
Ok, I will follow your advice then.